File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    map<char,int> countChar(const string& s) {
        map<char, int> charCount;
        for(int i = 0; i < s.size(); i++) {
            if(charCount.find(s[i]) != charCount.end()) {
                charCount[s[i]]++;
            } else {
                charCount.insert(pair<char,int>(s[i],1));
            }
            
        }
        return charCount;
    }

    bool isAnagram(string s, string t) {
        if(s.size() != t.size()) {
            return false;
        }

        map<char,int> count1 = countChar(s);
        map<char,int> count2 = countChar(t); 

        return count1 == count2;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        if len(s) != len(t):
            return False

        ss = sorted(s)
        tt = sorted(t)

        return ss == tt
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        